specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -3446,52 +3992,55 @@
           {
             "name": "name",
             "in": "path",
-            "description": "Name of the app.",
-            "required": true,
-            "type": "string"
-          },
-          {
-            "name": "id",
-            "in": "path",
-            "description": "Deployment ID.",
+            "description": "Name of web app",
             "required": true,
             "type": "string"
           },
           {
             "name": "instanceId",
             "in": "path",
-            "description": "ID of a specific scaled-out instance. This is the value of the name property in the JSON response from \"GET api/sites/{siteName}/instances\"",
+            "description": "Id of web app instance",
             "required": true,
             "type": "string"
           },
           {
-            "$ref": "#/parameters/subscriptionIdParameter"
-          },
-          {
+            "name": "MSDeploy",
+            "in": "body",
+            "description": "Details of MSDeploy operation",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/MSDeploy"
```

Comment:
are you talking about the "schema" block?  If so, I don't understand your comment.  The only thing in the "schema" is
`            "schema": {
              "$ref": "#/definitions/MSDeploy"
            }`